trailblazer/apps/slurm/api.py

```diff
@@ -1,10 +1,50 @@
+import subprocess
+from pathlib import Path
 from typing import Callable, Dict, List, Optional
 
 from trailblazer.apps.slurm.models import SqueueResult
 from trailblazer.apps.slurm.utils import formatters
 from trailblazer.constants import FileFormat, SlurmJobStatus, TrailblazerStatus
 from trailblazer.exc import EmptySqueueError
-from trailblazer.io.controller import ReadStream
+from trailblazer.io.controller import ReadFile, ReadStream
+
+
+def _get_squeue_jobs_flag_input(slurm_job_id_file_content: Dict[str, List[str]]) -> str:
+    """Return a string of comma separated SLURM job ids to be used as input for squeue jobs flag."""
+    job_ids: List[str] = []
+    for slurm_job_ids in slurm_job_id_file_content.values():
+        [job_ids.append(str(job_id)) for job_id in slurm_job_ids]
+    return ",".join(job_ids)
+
+
+def get_slurm_squeue_output(slurm_job_id_file: Path, use_ssh: bool = False) -> str:
+    """Return squeue output from ongoing analyses in SLURM."""
+    slurm_job_id_file_content: Dict[str, List[str]] = ReadFile.get_content_from_file(
+        file_format=FileFormat.YAML, file_path=slurm_job_id_file
+    )
+    slurm_jobs: str = _get_squeue_jobs_flag_input(
+        slurm_job_id_file_content=slurm_job_id_file_content
+    )
+    squeue_commands: List[str] = [
+        "squeue",
+        "--jobs",
+        slurm_jobs,
+        "--states=all",
+        "--format",
+        "%A,%j,%T,%l,%M,%S",
+    ]
+    if use_ssh:
+        squeue_commands = ["ssh", "hiseq.clinical@hasta.scilifelab.se"] + squeue_commands
+        return (
+            subprocess.check_output(
+                squeue_commands,
+                universal_newlines=True,
+            )
+            .decode("utf-8")
```

Comment:
```suggestion
            .decode(UTF8CONSTANT)
```
seen this come around a couple of times as hard coded string in the code

Reply:
Fixed